app/models/order_item.rb

```diff
@@ -1,9 +1,11 @@
 class OrderItem < ApplicationRecord
   belongs_to :order
   belongs_to :item
+  belongs_to :store
+  delegate :store, to: :item
```

Comment:
Since an order_item belongs to an item, delegate tells an instance of order_item to go through it's item in order to find the store.

Here's documentation for delegate: https://apidock.com/rails/Module/delegate

In future stories, we'll create a rake task to save the store_id on the order_item itself.